### PR TITLE
Add a descriptor for non-proxy functions

### PR DESCRIPTION
**Pull request by teytaud**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context / Related issue

We need descriptors for characteristics which are critical for describing the problem. Experiments have clearly shown that this descriptor is much needed.

Problems in which we optimize a proxy of the objective function instead of optimizing the real objective function are ubiquitous:
- train error or valid error vs test error
- distribution shift
- reality gap in RL
- IQA versus human opinion.

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the [**CONTRIBUTING**](https://facebookresearch.github.io/nevergrad/contributing.html) document and completed the CLA (see [**CLA**](https://facebookresearch.github.io/nevergrad/contributing.html#contributor-license-agreement-cla)).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

**Comment by jrapin** [comment]
Please explain what this is about, I don't understand this and I would avoid this is possible.

**Comment by teytaud** [comment]
> Please explain what this is about, I don't understand this and I would avoid this is possible.

Fully described now

**Comment by jrapin** [comment]
ok, let's discuss this next time we talk, not sure it's the best way of doing it, but not sure either what would be a better option

**Review by jrapin — APPROVED** [review]

File: nevergrad/parametrization/utils.py [hunk 1]
```diff
@@ -17,19 +17,38 @@
...
+        contains a softmax.
+    deterministic_function: bool
+        whether the objective function is deterministic.
+    non_proxy_function: bool
+        whether the objective function is not a proxy of a more interesting objective function.
+    continuous:
+        whether the domain is entirely continuous.
+    metrizable:
+        whether the domain is naturally equipped with a metric.
+    ordered:
```
Comment (jrapin) [inline nevergrad/parametrization/utils.py:35]:
```suggestion
    continuous: bool
        whether the domain is entirely continuous.
    metrizable: bool
        whether the domain is naturally equipped with a metric.
    ordered: bool
```

**Pull request merged**